include/openenclave/internal/calls.h

```diff
@@ -483,8 +483,17 @@ oe_result_t oe_ocall(uint16_t func, uint64_t arg_in, uint64_t* arg_out);
 **==============================================================================
 */
 
-#define OE_SYSCALL_OCALL_FUNCTION_TABLE_ID 0
-#define OE_SYSCALL_ECALL_FUNCTION_TABLE_ID 0
+#define OE_INTERNAL_OCALL_FUNCTION_TABLE_ID 0
+#define OE_INTERNAL_ECALL_FUNCTION_TABLE_ID 0
+
+#define OE_SYSCALL_OCALL_FUNCTION_TABLE_ID 1
+#define OE_SYSCALL_ECALL_FUNCTION_TABLE_ID 1
```

Comment:
These table ID's are being added in a platform-agnostic header, but are currently only used for SGX.

Reply:
I understand we have to rely on host for certain syscalls, but what's the usage of `OE_SYSCALL_ECALL_FUNCTION_TABLE_ID`?

Reply:
**OE_SYSCALL_ECALL_FUNCTION_TABLE_ID** is pre-existing. It corresponds to the EDL file under here:

```
common/syscall.edl
```

These are not SGX-specific though

Reply:
By the way, this header is not platform agnostic. It contains many SGX specific ECALL/OCALL definitions. I'll clean that up in part II.

Reply:
I missed out on that; I had only looked at the file path, which doesn't have `sgx` in it.